gtfsrt_events_vp_current_status/Program.cs

```diff
@@ -17,7 +17,7 @@ private static void Main()
             ServiceBase.Run(ServicesToRun);
 #else
             var service = new gtfsrt_events_vp_current_status_service();
-            service.StartGTFSRealtimeService();
+            gtfsrt_events_vp_current_status_service.StartGTFSRealtimeService();
```

Comment:
Do you still need the previous `var service` line in that case?

Reply:
No, the line can be deleted